libs/server/Objects/Hash/HashObjectImpl.cs

```diff
@@ -430,7 +430,7 @@ private void IncrementIntegerOrFloat(HashOperation op, byte* input, int length,
                     }
                     else
                     {
-                        var errorMessage = Encoding.ASCII.GetBytes("-ERR field value is not a number\r\n");
+                        ReadOnlySpan<byte> errorMessage = "-ERR field value is not a number\r\n"u8;
```

Comment:
u8 literals aren't collection expressions - they don't require target-typing as their natural type is already ROS<byte> :)